modules/phenyl-api-explorer/src/PhenylApiExplorer.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-env node */
+// @flow
+import fs from 'fs'
+import path from 'path'
+import ejs from 'ejs'
+import type {
+  FunctionalGroup,
+  EncodedHttpResponse,
+} from 'phenyl-interfaces'
+
+export type ExplorerParams = {|
+  path: string,
+|}
+
+export default class PhenylApiExplorer {
+  functionalGroup: FunctionalGroup;
+  path: string;
+  handler: any;
+  html: string;
+
+  constructor (functionalGroup: FunctionalGroup, params: ExplorerParams) {
+    this.functionalGroup = functionalGroup
+    this.path = params.path
+    this.handler = this.handler.bind(this)
+    this.html = this.getClientHTML()
+  }
+
+  shallowMap (obj: Object, fn: (any, string, Object) => any): Object {
```

Comment:
Is this just a utility function? If so, why don't you separate it into a plain function?